docs/start/install.md

```diff
@@ -55,6 +55,17 @@ pip install -e "python[all_hip]"
 Note: Please refer to [the CPU environment setup command list](../references/cpu.md#install-from-source)
 to set up the SGLang environment for running the models with CPU servers.
 
+Note: For Ascend NPUs, do the following instead, and more info can be found [here](../references/ascend_npu.md#installing-sglang-from-source)
```

Comment:
This is not needed anymore. I already put a placeholder for you and referenced in `install.md`.
https://github.com/sgl-project/sglang/blob/2449a0afe246d096f58e86c6b5f5563a63598cf4/docs/get_started/install.md?plain=1#L6